databroker/_core.py

```diff
@@ -1119,7 +1119,14 @@ def add_filter(self, **kwargs):
         :meth:`Broker.clear_filters`
 
         """
-        self.filters.append(dict(**kwargs))
+        for k,v in kwargs.items():
```

Comment:
It is better to remind users that there are existing filter names. Just keep them aware of this.